src/Microsoft.Health.Abstractions/Features/Events/IEventGridPublisher.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Messaging.EventGrid;
+
+namespace Microsoft.Health.Abstractions.Features.Events
+{
+    /// <summary>
+    /// Abstraction for the Azure.Messaging.EventGridPublisherClient
+    /// </summary>
+    public interface IEventGridPublisher
+    {
+        /// <summary>
+        /// SendEventAsync
+        /// </summary>
+        /// <param name="eventGridEvent">EventGridEvent</param>
+        /// <param name="cancellationToken">CancellationToken</param>
+        /// <returns>Task</returns>
+        public Task<Response> SendEventAsync(
+            EventGridEvent eventGridEvent,
```

Comment:
Is "EventGridEvent" included in this PR?

Reply:
This EventGridEvent is from Azure lib